pkg/registry/pod/rest.go

```diff
@@ -59,6 +59,14 @@ func (podStrategy) ResetBeforeCreate(obj runtime.Object) {
 	}
 }
 
+// PrepareForUpdate clears fields that are not allowed to be set by end users on update.
+func (podStrategy) PrepareForUpdate(obj, old runtime.Object) {
+	newPod := obj.(*api.Pod)
+	oldPod := old.(*api.Pod)
+	newPod.Status = oldPod.Status
+	//FIXME: can Spec.Host be updated?
```

Comment:
That's a good point.  @smarterclayton should we validate first?  Or should we allow Prepare to return errors?  Or should we intentionally copy Host over and let it fail in validation?